.pre-commit-config.yaml

```diff
@@ -43,5 +45,5 @@ repos:
           - --max-line-length=120
           - --min-public-methods=0
           - --good-names=q,f,fp,i,e
-          - --disable=E0401,W1201,W1203,W0603,C0114,C0115,C0116,C0411
+          - --disable=E0401,W1201,W1203,W0603,C0114,C0115,C0116,C0411,W0107,C0209
```

Comment:
should we move this into its own PR ? just to keep purpose separated ?

Reply:
Yes pls I just created a PR for it #83

Reply:
reverted